onnxruntime/core/providers/openvino/ov_versions/capability.cc

```diff
@@ -115,7 +117,13 @@ std::vector<std::unique_ptr<ComputeCapability>> GetCapability::Execute() {
     openvino_ep::BackendManager::GetGlobalContext().is_wholly_supported_graph = true;
 
   } else {  // unsupported_nodes_idx.empty()
-   
+
+  #if defined(OPENVINO_DISABLE_GRAPH_PARTITION) // disable_graph_partition at build time
```

Comment:
is there a reason it's a build option rather than a runtime option?

Reply:
Hi George. so, we didn't want to have this option at runtime because we think this option is not for all the users. It would be ideal to have it just at build time. so that a particular user/customer who really needs this option for his use case, can go through this route (Build time).

Also to introduce this option at runtime needs changes at lot of places (touch base on different API's) and we think this level of changes are not required for this feature.